docs/app/components/migrate-link/migrate-link.component.html

```diff
@@ -0,0 +1,43 @@
+<ng-template #popTemplate>
+    <table class="migrate-popover">
+        <tbody>
+            <tr>
+                <th colspan="2"><a [attr.href]="'https://pages.github.houston.softwaregrp.net/Inner-Source-Incubating/ux-aspects-universal/angular-storybook/?path=/docs/' + migration.href">UX Aspects Universal: {{ migration.title }}</a></th>
+            </tr>
+            <tr>
+                <td class="universal-info" colspan="2">
+                    UX Aspects Universal components are lightweight and fully style-encapsulated,
+                    which means you can drop them into a page without disrupting other components.
+                    Benefits include a dark theme and right-to-left language support.
+                    <a href="https://pages.github.houston.softwaregrp.net/Inner-Source-Incubating/ux-aspects-universal/angular-storybook/?path=/docs/welcome-migrating-from-ux-aspects--page">More...</a>
+                </td>
+            </tr>
```

Comment:
I don't think this warrants being a table since you are just overriding the columns.